commands.go

```diff
@@ -1113,7 +1119,11 @@ func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores bo
 }
 
 func (c *commandable) ZRangeByScore(key string, opt ZRangeByScore) *StringSliceCmd {
-	return c.zRangeByScore(key, opt, false)
+	return c.zRangeByScore(key, opt, false, false)
+}
+
+func (c *commandable) ZRangeByLex(key string, opt ZRangeByScore) *StringSliceCmd {
```

Comment:
I think breaking API in this case is not worth it. Besides that I prefer to have `ZRangeByLex` and `ZRangeByScore` rather than `ZRangeByOpts`. It reads better and compiler helps finding typos in the name :)